src/network/NetworkController.ts

```diff
@@ -94,7 +99,7 @@ export class NetworkController extends BaseController<
   NetworkConfig,
   NetworkState
 > {
-  private ethQuery: any;
+  private ethQuery: EthQuery | null;
```

Comment:
`ethQuery` can be null because it is not initialized in the constructor (as far as I could tell).